iidm/iidm-api/src/main/java/com/powsybl/iidm/network/ShuntCompensator.java

```diff
@@ -117,7 +119,7 @@
      * Depends on the working variant.
      * @see VariantManager
      */
-    int getSectionCount();
+    OptionalInt getSectionCount();
```

Comment:
This is a strong breaking change, do not forget to deal with the backward compatibility of groovy scripts at least. We have to ask the community about the potential impacts.